pre_commit/commands/autoupdate.py

```diff
@@ -79,8 +79,10 @@ def _check_hooks_still_exist_at_rev(
     hooks_missing = hooks - {hook['id'] for hook in manifest}
     if hooks_missing:
         raise RepositoryCannotBeUpdatedError(
-            f'Cannot update because the tip of HEAD is missing these hooks:\n'
-            f'{", ".join(sorted(hooks_missing))}',
+            f'Cannot update because the latest tag is missing these hooks:\n'
```

Comment:
hmmm but this can also show up when using `--bleeding-edge` so this is subtly wrong too 🤔 

how about "tip of HEAD" -> "update target" and not adding the new sentence

Reply:
ah, you're correct -- didn't think about that